source/MaterialXRenderGlsl/GLFramebuffer.cpp

```diff
@@ -127,6 +127,34 @@ GLFramebuffer::~GLFramebuffer()
     }
 }
 
+void GLFramebuffer::resize(unsigned int width, unsigned int height)
+{
+    if (width * height <= 0)
+    {
+        return;
+    }
+    if (width != _width || _height != height)
```

Comment:
This is probably the only logic change. Allow for color and depth buffers to be resized by resizing the textures.